### PR TITLE
Recipes list

### LINKED ISSUE #4: [3pts] Lucas: Public recipe list
- [x] Set up the database migration
- [x] Set up the model
- [x] Set up the controller
- [x] Create the view

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
# Project Requirements ⚙️ 
- Should display a list of recipes created by the logged-in user as in the wireframe.
- Should lead to recipe details.
- If the user is the recipe owner, it should allow the user to delete it.

### CODE REVIEW

File: config/routes.rb
Comment (vechicin) [inline config/routes.rb:4]: Make sure the routes are nested within the `users` resources, we need the recipes to be dependent on the user who signs in

File: app/views/layouts/application.html.erb
Comment (vechicin) [inline app/views/layouts/application.html.erb]: Make sure the `Sign Out` button is only visible IF the user is logged in, right now it is visible at all times.

File: app/views/recipes/index.html.erb
Comment (vechicin) [inline app/views/recipes/index.html.erb]: [OPTIONAL]
I think it'll be better UX if we change this button's text to Delete this recipe or Remove this recipe or simply Delete or Remove

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hello @Lucas-Masaba ✋ 
Really good work! Most of the features work perfectly and the codebase looks really cleaned and organized!
## STATUS: CHANGES REQUIRED ♻️ 
There are a couple of changes that need to be implemented before the approval of this pull request.

**Take a look at the changes below**

Comments marked as [OPTIONAL] won't prevent this request from being approved, however I strongly encourage you to implement these suggestions as they will make your project better.

File: app/views/layouts/application.html.erb
Comment (Lucas-Masaba) [reply app/views/layouts/application.html.erb]: Thank you for pointing that out. I have looked into it and fixed it.

File: app/views/recipes/index.html.erb
Comment (Lucas-Masaba) [reply app/views/recipes/index.html.erb]: Ah, yes. A good observation. Thank you for that.

File: config/routes.rb
Comment (Lucas-Masaba) [reply config/routes.rb:4]: Thanks for that

**Review by vechicin — APPROVED** [review]
# Hello @Lucas-Masaba ✋ 
Fantastic work, that means that...
## STATUS: APPROVED 🎉 
Awesome job implementing the required changes!

**There is nothing else to say other than, is time to merge it** 🕵️

**Pull request merged**
